doc/api/typescript.md

```diff
@@ -81,6 +81,7 @@ but we recommend version 5.8 or newer with the following `tsconfig.json` setting
 ```json
```

Comment:
I think this will have to be `jsonc` to be formatted correctly?

Reply:
I don't think so. I see other uses of `json` with comments, like this:

https://github.com/nodejs/node/blob/a344300bfa146fce9d1f20a71b073263a4720eab/doc/api/report.md#L608-L612